server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala

```diff
@@ -36,5 +36,6 @@ class CreateInteractiveRequest {
   var queue: Option[String] = None
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
+  var scalaVersion: Option[String] = None
```

Comment:
I think we agree on the ultimate goal to support 2 Spark versions in runtime. However, we have different opinions on milestones. 
1. Support 2 Spark versions in a single assembly.
2. Support 2 Spark versions in runtime.

With milestone 1, we can release a single Livy assembly for different Scala and Spark version. This addresses the immediate need for my team.

Milestone 2 requires a lot of work. We need to think about how the configuration is laid-out (e.g. specify multiple SPARK_HOME). For Spark 1.6, how do we specify unofficial Scala binding per SPARK_HOME. How do users pick a specific Spark version with a specific Scala binding for batch, interactive and rsc. I don't think we have answers for these questions and I prefer to address milestone 1 first.

> I could remove this field, but it will lose the ability to use Spark with unofficial Scala buildings.
We can still support unofficial Scala buildings by changing it to a LivyConf.